packages/cli/src/commands/generate/templates/cell/test.js.template

```diff
@@ -1,10 +1,31 @@
-import React from 'react';
-import { fireEvent, cleanup } from '@testing-library/react'
+import { render, cleanup } from '@testing-library/react'
 
-import ${pascalName}Cell from './${pascalName}Cell'
+import { Loading, Empty, Failure, Success } from './${camelName}Cell'
```

Comment:
The cell will be created with a PascalCase filename like "AdminUsers" so this should be `pascalName`

Reply:
roger that. fixed.